docs/guides/cookbook.ipynb

```diff
@@ -319,29 +319,198 @@
+    "        examples[\"tokens\"].append(words)\n",
+    "    return examples\n",
+    "\n",
+    "# Get the prediction from our fine-tuned distilbert\n",
+    "def make_predictions(examples):\n",
+    "    examples[\"prediction\"] = classifier(examples[\"text\"], aggregation_strategy=\"first\")\n",
+    "    return examples\n",
+    "\n",
+    "# Add tokens and predictions\n",
+    "ag_news_prepared = ag_news.take(n)\\\n",
+    "    .map(make_tokens, batched=True)\\\n",
+    "    .map(make_predictions, batched=True, batch_size=32)\n",
+    "\n",
+    "# Make Rubrix records\n",
+    "records = []\n",
+    "for idx, example in tqdm(enumerate(ag_news_prepared), total=n, desc=\"Making records\"):\n",
+    "    record = rb.TokenClassificationRecord(\n",
+    "        text=example[\"text\"],\n",
+    "        tokens=example[\"tokens\"],\n",
+    "        prediction=[(p[\"entity_group\"], p[\"start\"], p[\"end\"]) for p in example[\"prediction\"]],\n",
+    "        prediction_agent=\"elastic/distilbert-base-cased-finetuned-conll03-english\",\n",
+    "        id=idx,\n",
+    "    )\n",
+    "    records.append(record)\n",
+    "\n",
+    "# Upload records to Rubrix\n",
+    "rb.log(records, \"ag_news_ner\")"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "After adding the records to Rubrix, we can now annotate them in the web app with the help of the predictions.\n",
+    "Afterward, we load them back into the notebook and fine-tune our model.\n",
+    "We chose the very efficient and light-weight [ELECTRA model](https://huggingface.co/google/electra-small-discriminator) to illustrate the procedure.\n",
+    "\n",
+    "Following training steps are a copy&paste of the excellent [Hugging Face How-to guides](https://huggingface.co/docs/transformers/tasks/token_classification).\n",
+    "Check them for more information about the training process."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {
+    "tags": []
+   },
+   "outputs": [],
+   "source": [
+    "from transformers import AutoTokenizer, DataCollatorForTokenClassification\n",
+    "\n",
+    "# Load the dataset from the web app and prepare it for training a Hugging Face transformer\n",
+    "agnews_ds = rb.load(\"ag_news_ner\", as_pandas=False).prepare_for_training()\n",
+    "\n",
+    "# Split it into a train and test set\n",
+    "agnews = agnews_ds.train_test_split()\n",
+    "\n",
+    "# We need to properly tokenize our data and align our labels\n",
+    "tokenizer = AutoTokenizer.from_pretrained(\"google/electra-small-discriminator\")\n",
+    "data_collator = DataCollatorForTokenClassification(tokenizer=tokenizer)\n",
+    "\n",
+    "def tokenize_and_align_labels(examples):\n",
+    "    tokenized_inputs = tokenizer(examples[\"tokens\"], truncation=True, is_split_into_words=True)\n",
+    "\n",
+    "    labels = []\n",
+    "    for i, label in enumerate(examples[f\"ner_tags\"]):\n",
```

Comment:
Just to double-check does our `prepare_for_training` method create a `ner_tags` column?

Also, why do we need to use `f"ner_tags"` instead of just `"ner_tags"`

Reply:
yes, the prepare_for_training creates a `ner_tags` column now, and no, the f is not necessary ... :)